Project structure changed, fastapi and mongodb added

lahim opened the pull request.

lahim committed:
Project structure changed, fastapi and mongodb added

ririshi commented on uaber-api/uaber/models.py:
```diff
@@ -0,0 +1,84 @@
+    name: str
+    vehicle: Vehicle
+    address: Address
+    check_in_loc: Optional[Location] = None
```
I think it would make more sense to treat check-ins as a separate class, as we would need to store drivers separately if we want to persist their profiles apart from their trips.

lahim committed:
Add driver action model

Change log:
- Driver action model added
- Driver action type added
- Driver check-in model added
- Driver check-out model added

Rav944 reviewed (commented):
everything works correctly !
